backend/app/services/adapters/marketplace_skill.py

```
@@ -0,0 +1,1084 @@
+class MarketplaceSkillService:
+    """Service for managing Skill Marketplace"""
+
+    # ==================== Category Management ====================
+
+    def list_categories(self, db: Session) -> SkillCategoryListResponse:
+        """List all skill categories with skill counts"""
+        categories = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "SkillCategory",
+                Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)